tools/doc/html.js

```diff
@@ -382,13 +382,19 @@ function buildToc({ filename, apilinks }) {
       node.children.push({ type: 'html', value: anchor });
     });
 
-    file.toc = unified()
-      .use(markdown)
-      .use(gfm)
-      .use(remark2rehype, { allowDangerousHtml: true })
-      .use(raw)
-      .use(htmlStringify)
-      .processSync(toc).toString();
+    if (toc !== '') {
+      file.toc = '<details id="toc" open><summary>Table of contents</summary>' +
```

Comment:
This PR also removes the capitalisation of `Contents`. That's fine with me, but I wanted to point it out in case it was not on purpose. Other occurrences uses the capitalization:

https://github.com/nodejs/node/blob/7794d36a372b7cb448f5a637765a8fe2e1431d95/doc/guides/cpp-style-guide.md#L6
https://github.com/nodejs/node/blob/4d3e87da3b4a8c5f369d024b10bd21a40761d381/doc/guides/releases.md#L8
https://github.com/nodejs/node/blob/860c18be8786533915ff0ae8ad949c7d5404b08a/doc/guides/writing-and-running-benchmarks.md#L3

Reply:
It was indeed on purpose.  [Our documentation style guide indicates the use of sentence case for document headings](https://github.com/nodejs/node/blob/835b85d5e992bc6a1b8b0a6d95068bb357d1fffc/doc/guides/doc-style-guide.md). This isn't exactly a heading (but the other things you point to are). However, the style guide also says to refer to Microsoft's style guide for things it (our style guide) doesn't cover, and [Microsoft's style guide recommends sentence case for situations like this](https://docs.microsoft.com/en-us/style-guide/text-formatting/using-type/use-sentence-style-capitalization).